frontend/src/lib/theme.ts

```diff
@@ -0,0 +1,38 @@
+const themeColors = {
+	chartcolors: {
+		dodgerBlue: '#2F80ED',
+		mediumOrchid: '#BB6BD9',
+		seaBuckthorn: '#F2994A',
+		seaGreen: '#219653',
+		turquoiseBlue: '#56CCF2',
+		festivalOrange: '#F2C94C',
+		silver: '#BDBDBD',
+		outrageousOrange: '#FF6633',
+		roseBud: '#FFB399',
```

Comment:
if most of them are constant right let's put them into https://github.com/SigNoz/signoz/tree/develop/frontend/src/constants

Reply:
Nice suggestion @palashgdev . Done.